### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by quarkytale**
Updated instructions on testing generated docs locally

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,9 +13,9 @@
         git clone https://github.com/gazebo-web/gazebosim-web-frontend
         cd web
 
-1. This repository comes with an example `setup.bash` file which works with the
+1. Add a `setup.bash` file which works with the
    API at https://staging-api.gazebosim.org. Edit that file with
-   your own Auth0 and backend information. Always remember to source this file
+   your own Auth0 and backend information i.e. the environment variables set while [installing the backend](https://github.com/gazebo-web/gazebosim-web-backend#install). Always remember to source this file
```
Comment (azeey) [inline README.md:18]: In my testing, I didn't have to set any environment variables in the frontend. Do you remember which variables you had to set?

Comment (mabelzhang) [reply README.md:18]: Is this the frontend.bash? I wasn't able to load the Docs page even after using that file :( Before I used it though, sometimes it worked. It's always been flakey for me...

Comment (azeey) [reply README.md:18]: @mabelzhang I encountered the issue you ran into with the page being empty. Rebuilding the docker image fixed it for me.

Comment (quarkytale) [reply README.md:18]: The variables set in point 6 in https://github.com/gazebo-web/gazebosim-web-backend#install.

This with following the install intructions is something that worked for me, though I didn't go back and forth to test dependencies, will update once I try again.

Comment (mabelzhang) [reply README.md:18]: Oh damn, I just realized while reading this diff that my local path in docker-compose.yml has been overwritten probably from a pull :sweat_smile: That's probably why it hasn't worked for me lately -_-

Updating that file actually fixed it for me and I didn't need to set the environment variables. But I'm not using the install instructions; I'm just using the Docker setup.

Comment (quarkytale) [reply README.md:18]: Ahh okay, I still need to try again. Meanwhile, these instructions definitely help when developing, so might be worth documenting.

**Review by azeey — APPROVED** [review]

**Pull request merged**